builtin/providers/openstack/resource_openstack_compute_instance_v2.go

```diff
@@ -262,42 +262,79 @@ func resourceComputeInstanceV2Read(d *schema.ResourceData, meta interface{}) err
 	d.Set("access_ip_v4", server.AccessIPv4)
 	d.Set("access_ip_v6", server.AccessIPv6)
 
-	host := server.AccessIPv4
-	if host == "" {
+	hostv4 := server.AccessIPv4
+	if hostv4 == "" {
 		if publicAddressesRaw, ok := server.Addresses["public"]; ok {
 			publicAddresses := publicAddressesRaw.([]interface{})
 			for _, paRaw := range publicAddresses {
 				pa := paRaw.(map[string]interface{})
 				if pa["version"].(float64) == 4 {
-					host = pa["addr"].(string)
+					hostv4 = pa["addr"].(string)
 					break
 				}
 			}
 		}
 	}
 
-	// If no host found, just get the first IP we find
-	if host == "" {
+	// If no host found, just get the first IPv4 we find
+	if hostv4 == "" {
 		for _, networkAddresses := range server.Addresses {
 			for _, element := range networkAddresses.([]interface{}) {
 				address := element.(map[string]interface{})
 				if address["version"].(float64) == 4 {
-					host = address["addr"].(string)
+					hostv4 = address["addr"].(string)
 					break
 				}
 			}
 		}
 	}
-	d.Set("access_ip_v4", host)
-	d.Set("host", host)
+	d.Set("access_ip_v4", hostv4)
+	d.Set("host", hostv4)
```

Comment:
OK, I'd be happy to submit another PR with them removed. Or unless you don't mind if it's done in this one?